contracts/polygon/connectors/aave-rewards/main.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity ^0.7.0;
+
+import { TokenInterface } from "../../common/interfaces.sol";
+import { Stores } from "../../common/stores.sol";
+import { Helpers } from "./helpers.sol";
+import { Events } from "./events.sol";
+
+abstract contract IncentivesResolver is Helpers, Events {
+
+    function claim(
+        address[] calldata assets,
+        uint256 amt,
+        bool stake,
+        uint256 getId,
+        uint256 setId
+    ) external payable returns (string memory _eventName, bytes memory _eventParam) {
+        uint _amt = getUint(getId, amt);
+
+        require(assets.length > 0, "invalid-assets");
+
+        _amt = incentives.claimRewards(assets, _amt, address(this), stake);
+
+        setUint(setId, _amt);
```

Comment:
Why aren't we using uint(-1) for 100% claiming. Also, I think the claiming should be 100% all the time so no need for amounts.

Reply:
any amount higher than maximum rewards will get replaced by maximum rewards. Sending uint(-1) will work in this case